src/services/penaltyDocuments.js

```diff
@@ -215,51 +227,80 @@ export default class PenaltyDocument {
 		const { Value } = body;
 		// may not need to remove this delete Value.paymentToken;
 		const newHash = hashToken(id, Value, Enabled);
+		let paidStatus = 'UNPAID';
+		const idList = [];
+		idList.push(id);
+		this.getPaymentInformation(idList)
+			.then((response) => {
+				if (response.payments !== null && typeof response.payments !== 'undefined' && response.payments.length > 0) {
```

Comment:
Just for future reference ```something != null``` (note the single equals) checks for both null and undefined, and if the expression is within the if condition then it doesn't really need that (because null and undefined are falsy) - the if statement could be simplified to ```if (response.payments && response.payments.length > 0)```